runbotics-orchestrator/src/main/java/com/runbotics/domain/Process.java

```diff
@@ -305,6 +265,32 @@ public void setSchedules(Set<ScheduleProcess> schedules) {
         this.schedules = schedules;
     }
 
+    public ZonedDateTime getLastRun() {
+        return lastRun;
+    }
+
+    public Process lastRun(ZonedDateTime lastRun) {
+        this.lastRun = lastRun;
+        return this;
+    }
```

Comment:
what is it for? imo function name doesn't describe it

Reply:
To remove.